libs/community/tests/integration_tests/llms/test_ipex_llm.py

```diff
@@ -86,3 +86,23 @@ def test_save_load_lowbit(model_id: str) -> None:
     )
     output = loaded_llm.invoke("Hello!")
     assert isinstance(output, str)
+
+@skip_if_no_model_ids
+@pytest.mark.parametrize(
+    "model_id",
+    model_ids_to_test,
+)
+def test_load_generate_gpu(model_id: str) -> None:
+    """Test valid call."""
+    llm = IpexLLM.from_model_id(
+        model_id=model_id,
+        model_kwargs={
+            "temperature": 0,
+            "max_length": 16,
+            "trust_remote_code": True,
+        },
+        device_map="xpu",
+    )
+    output = llm.generate(["Hello!"])
+    assert isinstance(output, LLMResult)
+    assert isinstance(output.generations, list)
```

Comment:
For embedding, I also treat device as an env variable: https://github.com/langchain-ai/langchain/blob/6bb9138b84bffe06e97e007082b0034e9666d4de/libs/community/tests/integration_tests/embeddings/test_ipex_llm.py#L16

Not sure whether we should unify here :)